.travis.yml

```diff
@@ -114,15 +117,16 @@ install:
 
 before_script:
     # Download the sample data for the build of the documentation.
-    - if [[ $SETUP_CMD == *build_docs* ]]; then python -c "import sunpy.data.sample"; fi
-    - if [[ $SETUP_CMD == *figure* ]]; then conda env create --file sunpy/tests/figure_tests_env_$PYTHON_VERSION.yml; source activate sunpy-figure-tests-$PYTHON_VERSION; fi
+    - if [[ $SETUP_CMD != *egg_info* ]]; then python -m "sunpy.data.sample"; fi
+    - if [[ $MAIN_CMD == py.test* ]]; then python setup.py build; fi
+    - if [[ $SETUP_CMD == *--figure* ]]; then conda env create --file sunpy/tests/figure_tests_env_$PYTHON_VERSION.yml; source activate sunpy-figure-tests-$PYTHON_VERSION; fi
```

Comment:
fyi you can directly feed the env file to ci-helpers

Reply:
oh rly?

Reply:
relatively recent addition, ``CONDA_ENVIRONMENT``